### ISSUE TITLE
Wercker is unstable in PRs for localization testing

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#3314]
from #3159 .

I do not like how wercker works .... some time restart works some time it does not.

https://app.wercker.com/#checkstyle/checkstyle/build/576de03eefc0c401000122d7?step=576de04c8c574e0001449dd4

Failed tests:
CheckerTest.testMultipleConfigs:633->BaseCheckTestSupport.verify:126->BaseCheckTestSupport.verify:134->BaseCheckTestSupport.verify:162 error message 0 expected:<...626812/file.java:1: [Missing a header - not enough lines in file].> but was:<...626812/file.java:1: [Header fehlt - Datei hat nicht genug Zeilen].>

it has problem with locale testing, it is so unstable in PRs. For now I did not noticed problem in master.

**Comment by romani** [comment]
Log from wercker from MEZk  user:
[wercker_maven_log.txt](https://github.com/checkstyle/checkstyle/files/333753/wercker_maven_log.txt)

@MEZk , can you reproduce this on local docker ?

**Comment by romani** [comment]
and now that problem is reproducible on master build - https://app.wercker.com/#checkstyle/checkstyle/build/57702cb06555a201000245f7?step=57702cbbe7a15a0001885710

we need to remove that build items back to Travis or other CI (shippable.com is suggested by MEZk) , I did some first steps - https://app.shippable.com/projects/577005b73be4f4faa56ad850

**Comment by MEZk** [comment]
@romani 

```
mvn clean integration-test failsafe:verify -DargLine='-Duser.language=de -Duser.country=DE'
```

Tests are failed at Shippable CI:
https://app.shippable.com/runs/577038e7d7b8e20c008a7e83

I think that problem is not with Wercker CI or Shippable CI. Both of them use Docker containers. From my point of view, the Docker images which are used by CIs do not have DE language packs.

**Comment by romani** [comment]
@MEZk , can you reproduce the same on local docker image ?  if not, and keeping the idea about language pack, can you remove language pack to reproduce the problem ?

**Comment by MEZk** [comment]
@romani 
Sorry for delay

I'm able to reproduce the issue. Here are the steps (docker must be installed):

1) build image
```
docker build -f myrepo/checkstyle .
```
2) run image in container
```
docker run myrepo/checkstyle mvn clean integration-test failsafe:verify -DargLine='-Duser.language=de -Duser.country=DE' 
```
[Log](https://github.com/checkstyle/checkstyle/files/335446/log.txt)

I faced the same problem on my local PC with Ubuntu 14.04, Maven 3.3.9 and JDK 1.8.

```
andreiselkin@andreiselkin ~ $ mvn -v
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T19:41:47+03:00)
Maven home: /usr/lib/apache/maven/apache-maven-3.3.9
Java version: 1.8.0_91, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-oracle/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.19.0-32-generic", arch: "amd64", family: "unix"
```

My [Dockerfile.txt](https://github.com/checkstyle/checkstyle/files/335437/Dockerfile.txt) based on [the dockerfile](https://github.com/carlossg/docker-maven/blob/40cbcd2edc2719c64062af39baac6ae38d0becf9/jdk-8/Dockerfile) that is used by our Wercker CI server with jdk8, Maven 3.3.9 and Checkstyle (running on Ubuntu 14.04).

P.S.
How to install docker: https://docs.docker.com/engine/installation/linux/ubuntulinux/

**Comment by romani** [comment]
closed in favour of #3317 .
